nimare/tests/test_estimator_performance.py

```diff
@@ -0,0 +1,268 @@
+import os
+
+import pytest
+from contextlib import ExitStack as does_not_raise
+import nibabel as nib
+
+from ..correct import FDRCorrector, FWECorrector
+from ..meta import ale, kernel, mkda
+from ..meta.utils import compute_kda_ma
+from ..transforms import mm2vox
+
+# set significance levels used for testing.
+ALPHA = 0.05
+BETA = 1 - ALPHA
```

Comment:
Not sure if it matters in this context, but beta is usually defined as the complement of power, not alpha. If the quantity you actually want is `1 - ALPHA`, let's call that the `TRUE_NEG` rate or something like that.

Reply:
I just wrote `1 - ALPHA` instead to side step interpretation, but I can rename to TRUE_NEG.